model/utils.go

```diff
@@ -26,10 +26,12 @@ import (
 )
 
 const (
-	LOWERCASE_LETTERS = "abcdefghijklmnopqrstuvwxyz"
-	UPPERCASE_LETTERS = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
-	NUMBERS           = "0123456789"
-	SYMBOLS           = " !\"\\#$%&'()*+,-./:;<=>?@[]^_`|~"
+	LOWERCASE_LETTERS     = "abcdefghijklmnopqrstuvwxyz"
+	UPPERCASE_LETTERS     = "ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+	NUMBERS               = "0123456789"
+	SYMBOLS               = " !\"\\#$%&'()*+,-./:;<=>?@[]^_`|~"
+	TIMESTAMP_UPPER_LIMIT = 30 //in second
```

Comment:
This name is a bit generic, and also seems to duplicate the definition in `web/webhook.go`. Intentional?